### PR TITLE
Set font families that were not specified.

### PR DESCRIPTION
**Pull request by HaudinFlorence**
When working on a theme editor extension, there were only there 3 texts items ("Code" in notebook toolbar, the name of the kernel and the switch label) that were not following the selected ui-font family.

## Code changes

Add font families in packages/ui-components/style in switch.css, base.css and toolbar.css

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,6 +108,7 @@
     "react": "^17.0.1"
   },
   "dependencies": {
+    "@jupyter/ydoc": "^0.2.2",
```
[the remark this answers is not in the document]
Comment (HaudinFlorence) [reply package.json]: There may be something I don't understand and that leads to this unwanted commited files. Everytime I rebase with master or start a new branch from master, I can't build jupyterlab anymore. I have an error message that I understood (maybe wrongly) as "jupyter/ydoc" is missing. So I re install it and then it builds... And so I have then commited the new version of yarn.lock and package.json.

**Comment by fcollonval** [comment]
bot please update snapshots

**Comment by fcollonval** [comment]
bot please update snapshots

**Review by fcollonval — APPROVED** [review]
Thanks @HaudinFlorence

**Pull request merged**